src/cleed++/BulkModel.hh

```
@@ -0,0 +1,43 @@
+ *
+ * @license GPL-3.0+ <http://spdx.org/licenses/GPL-3.0+>
+ *
+ *********************************************************************/
+
+/*! \file
+ *
+ * Provides a CLEED bulk model class.
+ */
+
+#ifndef __BulkModel_hh__
+#define __BulkModel_hh__
```

Clang (14.0.6)

[warning] declaration uses identifier '__BulkModel_hh__', which is a reserved identifier

(bugprone-reserved-identifier)